### PR TITLE
fixed issues with efficency calculation

### PR DESCRIPTION
**Pull request by fibnumb**

### CODE REVIEW

**Comment by raymondEhlers** [comment]
+test

**Comment by raymondEhlers** [comment]
+test

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx [hunk 1]
```diff
@@ -366,86 +398,116 @@ void AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::DoJetLoop()
     TString groupname;
     AliJetContainer* jetCont = 0;
     TIter next(&fJetCollArray);
-
-	TClonesArray *triggerpatches = dynamic_cast<TClonesArray *>(InputEvent()->FindListObject("EmcalTriggers"));
-	if(!triggerpatches)
-		AliErrorStream() << "Trigger patch container EmcalTriggers not found in task " << GetName() << std::endl;
-
+    
+    TClonesArray *triggerpatches = dynamic_cast<TClonesArray *>(InputEvent()->FindListObject("EmcalTriggers"));
```
Comment (mfasDa) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx]: Normally the AliAnalysisTaskEmcal takes care of reading objects from the event. Why is this needed?

**Comment by fibnumb** [comment]
At the very least when I run my analysis locally nothing with the trigger information is filled unless I recast this.

**Comment by raymondEhlers** [comment]
In general, I would recommend leaving the PR open and then pushing changes to the same branch. It will update the PR and it will be easier to keep track when issues are resolved and can be merged

**Comment by fibnumb** [comment]
ok, I didn't realize that updating my fork automatically closes my PR

**Comment by raymondEhlers** [comment]
Updating the branch shouldn't close the PR - it should just update it. Perhaps deleting the branch could do that? In any case, I wouldn't recommend deleting it
